subprojects/build-init/src/main/resources/org/gradle/buildinit/tasks/templates/library-versions.properties

```diff
@@ -4,8 +4,8 @@ groovy=2.5.12
 guava=29.0-jre
 junit-jupiter=5.6.2
 junit=4.13
-kotlin=1.4-M2-mt
```

Comment:
Ha! that won't fly. The automated process should be selecting stable versions only I suppose.

Reply:
Followed up: https://github.com/gradle/gradle/pull/13648